aeon/dj_pipeline/utils/streams_maker.py

```diff
@@ -15,8 +16,8 @@
 # schema_name = f'u_{dj.config["database.user"]}_streams'  # for testing
 schema_name = get_schema_name("streams")
 
-STREAMS_MODULE_NAME = "streams"
-_STREAMS_MODULE_FILE = Path(__file__).parent.parent / f"{STREAMS_MODULE_NAME}.py"
+STREAMS_MODULE_NAME = "aeon_streams"
+_STREAMS_MODULE_FILE = Path(__file__).parent.parent / "streams.py"
```

Comment:
the `STREAMS_MODULE_NAME` is not used anymore? instead, we are hardcoding `streams.py`?

Reply:
`STREAMS_MODULE_NAME` is used elsewhere to invoke virtual modules like

`streams = dj.VirtualModule("streams", streams_maker.STREAMS_MODULE_NAME)`

I think it can be hard-coded as the file name won't change.

Reply:
If that's the case, wouldn't it be:

```
STREAMS_MODULE_NAME = schema_name
```

The usage of `STREAMS_MODULE_NAME` is now a bit unclear

Reply:
@ttngu207. Yeah, that's better.